cec-mini-kb.cpp

```diff
@@ -267,7 +295,7 @@ int main(int argc, char* argv[])
 	}}
 
 	// Open a connection to the zeroth CEC device
```

Comment:
// Open a connection to the target CEC device

Reply:
my misstake